src/api/kotlin/xyz/wagyourtail/unimined/api/minecraft/PatchProviders.kt

```diff
@@ -334,6 +336,41 @@ interface PatchProviders {
         }
     }
 
+    /**
+     * enables the FoxLoader patcher.
+     * @since 1.4.0
+     */
+    fun foxLoader() {
```

Comment:
In a perfect world, I would like these to go in their own interface, since they aren't compatible with most Providers.

Reply:
agreed, that could be doable if you change the type returned by reIndev.

Reply:
That would require me to change what type the `minecrafts` map is in `UniminedExtensionImpl` because `MinecraftConfig` would not implement `ReIndevPatchProviders`. So it would be inaccessible from Gradle.

Reply:
yes